db/index.js

```diff
@@ -9,7 +9,8 @@ db.run(
         categoryId INTEGER NOT NULL,
         date DATE NOT NULL,
         vendor STRING NOT NULL,
-        amount REAL NOT NULL
+        amount REAL NOT NULL,
+        file_name STRING
```

Comment:
db convention is to use camelcase.  filePath would be more appropriate since it would contain path as well